### DISCUSSION TITLE
What is your source on JLTP Levels of Kanji?

### DISCUSSION THREAD

**Discussion by timo-a** [birchill/10ten-ja-reader#1974] [Q&A]
I just noticed that jisho.org, which uses [Jonathan Waller‘s JLPT Resources](http://www.tanos.co.uk/jlpt/) as a source (see footer [here](https://jisho.org/search/%E9%9B%A3)) assigns JLPT level 3 to '難' whereas 10ten has it at level 2. So 10ten must be using a different data source. May I ask which data source that is?

**Comment by birtles** [comment]
Hi! We're using the data from the KANJIDIC project as described here: http://www.edrdg.org/wiki/index.php/KANJIDIC_Project

Unfortunately that data is from the old JLPT. The new JLPT does not have a fixed set of kanji per level. At one point I created an updated mapping to the new levels based on comparison with the old data and 漢検 levels but I've never had enough confidence in it to switch it on by default.

I'm not sure what Jonathan Waller's data is based on but I assume it's a similar "guess".

I'd really like to OCR a series of past JLPT tests and try to create a more evidence-based mapping but I've never quite had time unfortunately.
